pkg/webui/account/views/update-password/index.js

```diff
@@ -28,25 +27,26 @@ import Form from '@ttn-lw/components/form'
 import Input from '@ttn-lw/components/input'
 import SubmitButton from '@ttn-lw/components/submit-button'
 import Checkbox from '@ttn-lw/components/checkbox'
-import Spinner from '@ttn-lw/components/spinner'
 
 import Message from '@ttn-lw/lib/components/message'
 import IntlHelmet from '@ttn-lw/lib/components/intl-helmet'
 
-import style from '@account/views/create-account/create-account.styl'
+import style from '@account/views/front/front.styl'
 
 import Yup from '@ttn-lw/lib/yup'
+import { selectApplicationSiteName } from '@ttn-lw/lib/selectors/env'
 import sharedMessages from '@ttn-lw/lib/shared-messages'
 import PropTypes from '@ttn-lw/lib/prop-types'
 
 const m = defineMessages({
+  updatePassword: 'Update password',
   newPassword: 'New password',
   oldPassword: 'Old password',
   passwordChanged: 'Password changed',
   revokeAccess: 'Revoke access',
-  logoutAllDevices: 'Log out from all end devices',
-  revokeWarning: 'This will revoke access from all logged in devices',
-  sessionRevoked: 'Session revoked',
+  logoutAllDevices: 'Sign out from all end devices',
```

Comment:
```suggestion
  logoutAllDevices: 'Sign out from all devices',
```

Reply:
This is not referring to end devices in the LoRaWAN sense.